src/sbt-test/sbt-cake/testrunner/build.sbt

```diff
@@ -18,9 +20,7 @@ val testrunner = (project in file("."))
     CakeDockerPlugin,
     CakeTestRunnerPlugin)
   .settings(
-    dependencyOverrides ++= Set(
-      "com.typesafe.akka" %% "akka-actor" % "2.4.20"
-    )
+    dependencyOverrides += "com.typesafe.akka" %% "akka-actor" % "2.4.20"
```

Comment:
Shouldn't this value be updated?

Reply:
Akka version didn't change in this PR so I guess not